src/main/java/peoplesoft/ui/regions/JobListPanel.java

```diff
@@ -0,0 +1,53 @@
+package peoplesoft.ui.regions;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import peoplesoft.commons.core.LogsCenter;
+import peoplesoft.model.job.Job;
+import peoplesoft.ui.UiPart;
+
+/**
+ * Panel containing the list of persons.
+ */
+public class JobListPanel extends UiPart<Region> {
+    private static final String FXML = "JobListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(JobListPanel.class);
+
+    @FXML
+    private ListView<Job> jobListView;
+
+    /**
+     * Creates a {@code JobListPanel} with the given {@code ObservableList}.
+     */
+    public JobListPanel(ObservableList<Job> jobList) {
+        super(FXML);
+        jobListView.setItems(jobList); // nullpointer
```

Comment:
wait what does the comment mean

Reply:
was a debug marker the other time lemme remove it